model/map-hot-rod/src/main/java/org/keycloak/models/map/common/AbstractHotRodEntity.java

```diff
@@ -17,6 +17,5 @@
 
 package org.keycloak.models.map.common;
 
-public interface Versioned {
-    int getEntityVersion();
+public interface AbstractHotRodEntity {
```

Comment:
I needed to mark all HotRodEntities with this interface in order to make this:  https://github.com/keycloak/keycloak/pull/8948/files#diff-1110818d9bc3b825b3324f67c809b938aebaf72c55fdbbf236736e17984b3591R376-R388 work.

Reply:
Here is a link directly to the file: https://github.com/keycloak/keycloak/blob/dfa6636aa94372e4c7ca889e122c2f581414b7ff/model/build-processor/src/main/java/org/keycloak/models/map/processor/GenerateHotRodEntityImplementationsProcessor.java#L376-L388

With the previous link, it should be enough to scroll to the `GenerateHotRodEntityImplementationsProcessor` file and click `Show diff`.